akka-sample-sharding-scala/build.sbt

```diff
@@ -1,33 +1,66 @@
 import com.typesafe.sbt.SbtMultiJvm.multiJvmSettings
 import com.typesafe.sbt.SbtMultiJvm.MultiJvmKeys.MultiJvm
 
-val akkaVersion = "2.6.0-M4"
+val AkkaVersion = "2.6.0-RC1"
+
+lazy val buildSettings = Seq(
+  organization := "com.lightbend.akka.samples",
+  scalaVersion := "2.12.10"
+)
+
+lazy val disciplineScalacOptions = Seq(
+  // not in 2.13
+  "-Ywarn-value-discard",
+  "-Ywarn-numeric-widen",
+  "-Yno-adapted-args",
+  // end
+  "-Xlog-reflective-calls",
+  "-Xlint",
+  "-deprecation",
+  "-Ywarn-dead-code",
+  "-Ywarn-inaccessible",
+  "-Ywarn-infer-any",
+  "-Ywarn-nullary-override",
+  "-Ywarn-nullary-unit",
+  "-Ywarn-unused:_",
+  "-Ypartial-unification",
+  "-Ywarn-extra-implicit"
+)
+
+lazy val commonScalacOptions =
+  disciplineScalacOptions ++ Seq(
+  "-deprecation",
+  "-feature",
+  "-unchecked",
+  "-encoding", "UTF-8"
+)
+
+lazy val commonJavacOptions = Seq(
+  "-Xlint:unchecked",
+  "-Xlint:deprecation"
+)
```

Comment:
Maybe just skip the discipline options in a sample?

Reply:
I like to have the security personally. What's the reason to skip?